emails/2023-06-23.md

```diff
@@ -0,0 +1,23 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+One of the joys of using [Discord](https://discord.gg/gqr7K4VNKe) for both our company and customer chat is seeing people's gamer tags. People show up with names like "CaptainStabs" or "Vecna". Some change it and some people roll with it. I like Discord because it's free for our customers. Come as you are!
+
+### Slowly Changing Dimension
+
+I wrote an opus on [Slowly Changing Dimension](https://www.dolthub.com/blog/2023-06-22-slowly-changing-dimension/). I know, I know you love to see "21 minute read" at the top of a blog. You don't need to read it unless you're curious. Just know, if you use Dolt, you get Slowly Changing Dimension on every column in your database. Your application can't even screw it up. 
+
+### Commit is Overloaded
+
+In Dolt, we have two types of commits: Transaction commits and Dolt commits. In standard SQL you commit a transaction. In Git, you make a commit when you want to permanently store the state of your repository. In Dolt, we support both of these concepts, overloading the term. It can be a bit confusing. Too make it even more confusing, you can [turn on `@@dolt_tranasction_commit`](https://www.dolthub.com/blog/2023-06-21-automatic-dolt-commits/) and get Dolt commits automatically for every sql transaction commit.
```

Comment:
```suggestion
In Dolt, we have two types of commits: Transaction commits and Dolt commits. In standard SQL you commit a transaction. In Git, you make a commit when you want to permanently store the state of your repository. In Dolt, we support both of these concepts, overloading the term. It can be a bit confusing. To make it even more confusing, you can [turn on `@@dolt_tranasction_commit`](https://www.dolthub.com/blog/2023-06-21-automatic-dolt-commits/) and get Dolt commits automatically for every sql transaction commit.
```